schema.go

```diff
@@ -149,17 +149,20 @@ func StreamingGetOldBuilds(ctx context.Context) (<-chan LogKeeperBuild, <-chan e
 			return
 		}
 
-		for cur.Next(ctx) {
+		for cur.Next(db.Context()) {
```

Comment:
Does the function signature need to accept a context anymore now that you are using the application context?

Reply:
Now that calls to `Next` take the application context I had this take the request context so we could check it in between calls to `Next`. This way when the request is cancelled we can still interrupt fetching the rest of the results from the db.